tests/mysql/tox.ini

```diff
@@ -26,19 +27,17 @@ setenv =
 passenv = *
 
 commands =
-    docker-compose stop
-    docker-compose rm -v -f
-    docker-compose pull db
+    docker-compose stop mysqldb
     bash -ec "if [ `uname` = 'Linux' ] && [ "{env:GITHUB_ACTIONS:false}" != "true" ]; then chcon -h system_u:object_r:bin_t:s0 sql; chcon -Rt svirt_sandbox_file_t sql; fi"
-    docker-compose up -d db
-    sleep 60
```

Comment:
This is the important change @rcbjBlueMars @nsahai @davexroth

Reply:
My next set of changes to the test suite are going to involve running the pipeline many times. So, I'm going to go ahead and merge this change.